ganache-blockchain-log.txt

```diff
@@ -0,0 +1,52 @@
+Ganache CLI v6.2.5 (ganache-core: 2.3.3)
```

Comment:
Was this file accidentally added? Do we actually need it?

Reply:
That was added by accident. Remove file and resubmit PR